packages/akamai/changelog.yml

```
@@ -1,7 +1,7 @@
 # newer versions go on top
 - version: "2.15.0"
   changes:
-    - description: Add tags.yml file.
+    - description: Add tags.yml file so that integration's dashboards and saved searches are tagged with "Security Solution", and will be shown in the Security Solution UI.
```

```suggestion
    - description: Add tags.yml file so that integration's dashboards and saved searches are tagged with "Security Solution" and displayed in the Security Solution UI.
```

nit: It has a compound verb so no comma is needed.